### ISSUE TITLE
TypeError: entry_points() got an unexpected keyword argument 'group'

### ISSUE THREAD

**Issue by alan1world** [jaraco/keyring#501]
**Describe the bug**
Calling keyring library causes a core dump.

```
Traceback (most recent call last):
  File "/home/human1/hg/2018/link_store_keep/package/link_store_v2.py", line 61, in setOutputChrome
    self.setOutput(self.plainTextEdit.toPlainText(), 'chrome', self.lineEdit.text(), self.lineEdit2.text())
  File "/home/human1/hg/2018/link_store_keep/package/link_store_v2.py", line 153, in setOutput
    self.build_mariadb(contents, db_config['user'], db_config['schema'], db_config['table'])
  File "/home/human1/hg/2018/link_store_keep/package/link_store_v2.py", line 99, in build_mariadb
    psswd = backup_mariadb.key_implement(dbase = dbase, usr = usr)
  File "/home/human1/hg/2018/link_store_keep/package/backup_mariadb.py", line 68, in key_implement
    if keyring.get_password(dbase, usr) == None:
  File "/usr/lib/python3.9/site-packages/keyring/core.py", line 55, in get_password
    return get_keyring().get_password(service_name, username)
  File "/usr/lib/python3.9/site-packages/keyring/core.py", line 32, in get_keyring
    init_backend()
  File "/usr/lib/python3.9/site-packages/keyring/core.py", line 83, in init_backend
    set_keyring(_detect_backend(limit))
  File "/usr/lib/python3.9/site-packages/keyring/core.py", line 101, in _detect_backend
    filter(limit, backend.get_all_keyring()),
  File "/usr/lib/python3.9/site-packages/keyring/util/__init__.py", line 22, in wrapper
    func.always_returns = func(*args, **kwargs)
  File "/usr/lib/python3.9/site-packages/keyring/backend.py", line 214, in get_all_keyring
    _load_plugins()
  File "/usr/lib/python3.9/site-packages/keyring/backend.py", line 198, in _load_plugins
    for ep in metadata.entry_points(group='keyring.backends'):
TypeError: entry_points() got an unexpected keyword argument 'group'
./link_store.sh: line 8: 14906 Aborted                 (core dumped) python link_store_v2.py
```

**Expected behavior**
Keyring library worked yesterday before system update. (Arch Linux updated it's package from 22.3 to 23.0 yesterday).

**Environment**

- OS: Arch Linux

```
$ pip list | grep keyring
...
keyring                       23.0.0
WARNING: Keyring is skipped due to an exception: entry_points() got an unexpected keyword argument 'group'

$ keyring --list-backends
...
  File "/usr/bin/keyring", line 33, in <module>
    sys.exit(load_entry_point('keyring==23.0.0', 'console_scripts', 'keyring')())
  File "/usr/lib/python3.9/site-packages/keyring/cli.py", line 135, in main
    return cli.run(argv)
  File "/usr/lib/python3.9/site-packages/keyring/cli.py", line 57, in run
    for k in backend.get_all_keyring():
  File "/usr/lib/python3.9/site-packages/keyring/util/__init__.py", line 22, in wrapper
    func.always_returns = func(*args, **kwargs)
  File "/usr/lib/python3.9/site-packages/keyring/backend.py", line 214, in get_all_keyring
    _load_plugins()
  File "/usr/lib/python3.9/site-packages/keyring/backend.py", line 198, in _load_plugins
    for ep in metadata.entry_points(group='keyring.backends'):
TypeError: entry_points() got an unexpected keyword argument 'group'
```

**Comment by cjc7373** [comment]
This seems to be a downstream issue. See: https://bugs.archlinux.org/task/69967

I've manually packed and installed python-importlib-metadata 3.7 and it works.

**Comment by alan1world** [comment]
Yes, I was reading the same comment.
It's been identified on the Arch site that it is cause by a dependency on importlib-metadata >=3.6. It was not highlighted on upgrade because the latest version available for Arch is importlib-metadata 3.4.
